services/web/server/src/simcore_service_webserver/projects/projects_handlers.py

```diff
@@ -158,6 +162,7 @@ async def list_projects(request: web.Request):
     project_type = ProjectTypeAPI(query["type"])
     offset = query["offset"]
     limit = query["limit"]
+    show_hidden = query["show_hidden"]
```

Comment:
Might be a stupid question but I would think that listing of projects requires the show_hidden, while loading the project should not require it.

Reply:
not at all. I even guess I know why you ask that... 

Unfortunately there are still small differences between the implementation of ``list_projects`` and ``get_project``.  The former uses ``openapi.yml`` specs to validate the request and produces ``query`` dict with values for required and defaults for optionals. In the latter there is NO validation function ... and we consider "explicitly" optional or required using using ``query.get(.)`` or ``query[.]``, resp. 

hopefully these "asymmetries" in the implementation will be resolved during the maintenance work